src/SFA.DAS.AssessorService.Application.Api/Controllers/OrganisationStandardController.cs

```diff
@@ -0,0 +1,48 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.AssessorService.Api.Types.Models;
+using SFA.DAS.AssessorService.Application.Api.Middleware;
+using SFA.DAS.AssessorService.Application.Api.Properties.Attributes;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Application.Api.Controllers
+{
+    [Authorize(Roles = "AssessorServiceInternalAPI")]
+    [Route("api/v1")]
+    public class OrganisationStandardController : Controller
+    {
+        private readonly ILogger<OrganisationStandardController> _logger;
+        private readonly IMediator _mediator;
+
+        public OrganisationStandardController(
+            ILogger<OrganisationStandardController> logger,
+            IMediator mediator
+        )
+        {
+            _logger = logger;
+            _mediator = mediator;
+        }
+
+        [HttpPost("organisationstandardversion", Name = "CreateOrganisationStandardVersion")]
+        [ValidateBadRequest]
+        [SwaggerResponse((int) HttpStatusCode.Created, Type = typeof(OrganisationResponse))]
+        [SwaggerResponse((int) HttpStatusCode.BadRequest, Type = typeof(IDictionary<string, string>))]
+        [SwaggerResponse((int) HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
+        public async Task<IActionResult> CreateOrganisationStandardVersion(
+            [FromBody] OrganisationStandardVersionOptInRequest request)
+        {
+            _logger.LogInformation("Received Create Organisation Standard Version Request");
```

Comment:
Nice creation of new controller to deal with new entity that wasn't used yet. I'm thinking about creating a tech debt ticket to move `GetEpaoRegisteredStandardVersions` from StandardVersionController to here as it seems to fit better? what do you think?

Reply:
I agree but i think there are more endpoints on the wrong controllers. Might be worth extending to the tech debt to a review of all the endpoints